network/basics/uri.py

```diff
@@ -129,6 +129,13 @@
         "Content-Type" along with your request with a value of "application/json".
     required: false
     default: null
+  headers:
+    description:
+      - A dictionary of headers you wish to pass to the URI.  This can compliment or be
+        used in place of the HEADER_ method.
```

Comment:
Typo: compliment -> complement.

It might be good to mention that if the same header shows up in both places, the HEADER_... version will override the one from `headers`.